src/github/middleware.ts

```diff
@@ -26,6 +30,7 @@ const withSentry = function (callback) {
 		try {
 			await callback(context);
 		} catch (err) {
+			logger.error(err, "Webhook Error");
```

Comment:
Should do `{...err}` to explode out all properties so Bunyan logs everything

Reply:
done. All other places where Sentry is used we already log a message.